sentry_sdk/_lru_cache.py

```diff
@@ -91,13 +89,6 @@ def __init__(self, max_size):
 
         self.hits = self.misses = 0
 
-    def __copy__(self):
```

Comment:
If you remove this method entirely, aren't we back at square one? I.e. the cache will get polluted by changes to a copy:
```
def test_mutate():
    cache = LRUCache(1)
    cache.set(0, 0)
    copy(cache).set(0, 1)
    assert cache.get(0) == 0
```

Reply:
No because [the call site](https://github.com/getsentry/sentry-python/pull/3882/files#diff-c935006d78056bc518dd6e898d31d972c130c63119df74edf122e2cf4a5733bcR255) is now using deepcopy.  Which targets the dunder deepcopy method which we leave as default.